src/components/EssayForm.js

```diff
@@ -0,0 +1,51 @@
+import React, { TextareaHTMLAttributes } from 'react';
+import Button from '@material-ui/core/Button'
+import { makeStyles } from '@material-ui/core/styles'
+
+const styles = {
+  button: {
+    
+  },
+};
+
+class EssayForm extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {
```

Comment:
Decide on a consistent style across all of the code.  I.e. either use "this." where applicable in all code or nowhere.  But don't use it sometimes and not in other files.